recipes/comboboxes/combobox_with_popover/combobox_with_popover.vue

```diff
@@ -94,6 +94,7 @@
           <div
             v-if="$slots.footer"
             ref="footer"
+            tabindex="0"
```

Comment:
We shouldn't be focusing a non interactive element. [focusout](https://developer.mozilla.org/en-US/docs/Web/API/Element/focusout_event) should still trigger when the input is focused as it captures anything inside of it as well this is the difference between [focusout](https://developer.mozilla.org/en-US/docs/Web/API/Element/focusout_event) and [blur](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur)

Reply:
I would suspect the problem is in this logic somewhere

```
const isComboboxStillFocused = comboboxRefs.some((ref) => {
        return this.$refs[ref]?.contains(e.relatedTarget);
      }) || this.visuallyHiddenClose;
```

Reply:
Final result after discussion

```
const isComboboxStillFocused = e.relatedTarget === null || comboboxRefs.some((ref) => {
        return this.$refs[ref]?.contains(e.relatedTarget);
      }) || this.visuallyHiddenClose;
```